.circleci/config.yml

```diff
@@ -355,6 +355,24 @@ workflows:
             branches:
               only:
                 - main
+      - deploy:
+          name: Deploy production to new namesapce
```

Comment:
small typo here!

Reply:
I've also spotted this on staging. Will update both.